proxy-manager/DOCS.md

```diff
@@ -82,8 +83,16 @@ cleared automatically_
   by taking a snapshot of just the Nginx Proxy Manager add-on, will not
   contain the data from this add-on. Please make sure you backup both
   the Nginx Proxy Manager and the MariaDB add-ons.
-
-## Changelog & Releases
+- As of Home Assitant Core 2021.07 unconfigured reverse proxies are blocked with a `400` http response.  
+  You may specify the default docker network to trust NPM, and all containers running on your host.
```

Comment:
Exposing your whole Docker network is bad practice, and that is not something I want to encourage or recommend in the documentation. Please remove it.